[2023-05-12T20:15:31.964-0500] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first.process manual__2023-05-13T01:14:45.908049+00:00 [queued]>
[2023-05-12T20:15:31.985-0500] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first.process manual__2023-05-13T01:14:45.908049+00:00 [queued]>
[2023-05-12T20:15:31.986-0500] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-05-12T20:15:32.127-0500] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): process> on 2023-05-13 01:14:45.908049+00:00
[2023-05-12T20:15:32.135-0500] {standard_task_runner.py:57} INFO - Started process 8006 to run task
[2023-05-12T20:15:32.148-0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'first', 'process', 'manual__2023-05-13T01:14:45.908049+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/first.py', '--cfg-path', '/tmp/tmp6ppn8vqx']
[2023-05-12T20:15:32.152-0500] {standard_task_runner.py:85} INFO - Job 6: Subtask process
[2023-05-12T20:15:32.316-0500] {task_command.py:410} INFO - Running <TaskInstance: first.process manual__2023-05-13T01:14:45.908049+00:00 [running]> on host adavid-home
[2023-05-12T20:15:32.584-0500] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='example@gmail.com' AIRFLOW_CTX_DAG_OWNER='adavid' AIRFLOW_CTX_DAG_ID='first' AIRFLOW_CTX_TASK_ID='process' AIRFLOW_CTX_EXECUTION_DATE='2023-05-13T01:14:45.908049+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-05-13T01:14:45.908049+00:00'
[2023-05-12T20:15:32.586-0500] {first.py:23} INFO - performing processing
[2023-05-12T20:15:32.587-0500] {python.py:183} INFO - Done. Returned value was: None
[2023-05-12T20:15:32.606-0500] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=first, task_id=process, execution_date=20230513T011445, start_date=20230513T011531, end_date=20230513T011532
[2023-05-12T20:15:32.687-0500] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2023-05-12T20:15:32.754-0500] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
